Documentation/developer-guide/contributing.md

```diff
@@ -0,0 +1,141 @@
+# Contributing to Autoscoper
```

Comment:
Let's keep this document at the root and include it instead of copying it.

To do so, in the file `Documentation/developer-guide/contributing.md`, you would have the following code:

<pre>
```{include} ../../CONTRIBUTING.md
```
</pre>